virtual-desktop/src/app/window-manager/mvd-window-manager/personalization-panel/personalization/personalization.component.html

```diff
@@ -31,22 +31,61 @@
     </file-drop>
 
   <label class="personalization-label">Color</label>
-  <div class="personalization-color-1">
-    <color-circle 
-      [color]="selectedColor" 
-      (onChangeComplete)="colorSelected($event.color)" 
-      [width]="380" 
-      [colors]='["#f44336", "#e91e63", "#9c27b0", "#673ab7", "#3f51b5", "#2196f3", 
-      "#03a9f4", "#00bcd4", "#009688", "#4caf50", "#8bc34a", "#cddc39", "#ffeb3b", "#ffc107", 
-      "#ff9800", "#252628", "#6d7176", "#f3f4f4"]'
-    ></color-circle>
+  <div class="personalization-color-1"> <!-- Color circle group -->
+    <div class="personalization-text" style="margin-left: -15px;">Select color</div>
+    <canvas #circle1 width="40" height="40" [ngClass]="{ 'personalization-circle selected': selectedCircle == 1,
```

Comment:
why did you make a canvas for every single circle? that's overkill and probably not very cpu efficient.
could just be css: https://www.w3schools.com/howto/howto_css_circles.asp

Reply:
Looks like most resources point to them being similar in performance, while DOM based rendering is described as being more memory intensive than the two surprisingly. CSS animations are also typically slower than canvas however text-based rendering suffers heavily with canvas. Both of those don't apply here though. Canvas consensus summed up well with
"
Fast. Really fast. Because an immediate mode system doesn't maintain its own model, your code is all that stands between you and the browser redrawing. The many layers of abstraction that slow operations down simply do not exist in the immediate mode world.
"
If you're okay, I'm okay
https://stackoverflow.com/questions/5882716/html5-canvas-vs-svg-vs-div
https://www.kirupa.com/html5/dom_vs_canvas.htm
https://www.reddit.com/r/html5/comments/2vnrvm/performance_of_the_canvas_vs_performance_of_the/